docs/source/onnx.rst

```diff
@@ -249,9 +249,41 @@ E.g.: ::
     out = model(*inputs)
     torch.onnx.export(model, inputs, 'loop_and_list.onnx', opset_version=11, example_outputs=out)
 
-Write PyTorch model in Torch way
+
+Type Annotations
 --------------------------------
+TorchScript only supports a subset of Python types. You can find more details about type annotation
+`here <https://pytorch.org/docs/stable/jit_language_reference.html#id8>`_.
+
+Due to optimization purposes, TorchScript only supports variables with single static types for script functions.
+By default, each variable is assumed to be Tensor. If an argument to a ScriptModule function is not Tensor,
+its type should be specified using MyPy-style annotations.
+
+::
+
+    import torch
+
+    class Module(torch.nn.Module):
+        def forward(self, x, tup):
+            # type: (int, Tuple[Tensor, Tensor]) -> Tensor
```

Comment:
Should it use Python3 typing style? (inline in code)

Reply:
The annotations? Yes. I'll add it.